lib/Elastica/Util.php

```diff
@@ -197,6 +197,6 @@ public static function convertRequestToCurlCommand(Request $request)
      */
     public static function debugEnabled()
     {
-        return defined('DEBUG') && DEBUG;
+        return defined('ELASTICA_DEBUG') && ELASTICA_DEBUG;
```

Comment:
I remember it was more a memory issue then a performance issue. CURLINFO_HEADER_OUT returns the full string as part of the response object. If you use bulk mode this can add up to quite a large object. CURLINFO_HEADER_OUT could also be set through the curl options.

The query time part should not have an affect and I think this is what is mostly needed.

Best would be to active the benchmark tests to see if there is a difference. They should work now again but are not enabled for travis.